products/neo4j.md

```diff
@@ -240,6 +248,11 @@ releases:
 
 Neo4j has separate policies for different versions.
 
+### [From January 2025]
+
+Neo4j Server adopted calendar versioning (CalVer). Earlier versions, such as Neo4j 4 and 5 used semantic versioning (SemVer).
```

Comment:
Is there a change to the support policy along with the versionning update ? Shouldn't this be documented in the "Latest Support Terms" paragraph ?

Reply:
I updated the description.